MoltenVK/MoltenVK/Utility/MVKFoundation.h

```diff
@@ -553,11 +553,10 @@ static void mvkRemoveAllOccurances(C& container, T val) {
 /** Selects and returns one of the values, based on the platform OS. */
 template<typename T>
 static constexpr const T& mvkSelectPlatformValue(const T& macOSVal, const T& iOSVal) {
-#if MVK_IOS_OR_TVOS
-	return iOSVal;
-#endif
 #if MVK_MACOS
 	return macOSVal;
+#else
+    return iOSVal;
```

Comment:
Does this work, or does this function need a rework? 

Are VisionOS values that might be used here the same as iOS values? So far, we're just using it for `MVKMTLFmtCaps`.

Reply:
might need rework later, so far focusing on build rather than functionality